src/Configuration/Config.Abstractions/src/ConfigurationExtensions.cs

```diff
@@ -36,6 +36,16 @@ public static string GetConnectionString(this IConfiguration configuration, stri
             return configuration?.GetSection("ConnectionStrings")?[name];
         }
 
+        /// <summary>
+        /// Shorthand for GetSection("ConnectionStrings")["DefaultConnection"].
+        /// </summary>
+        /// <param name="configuration">The configuration.</param>
+        /// <returns></returns>
+        public static string GetDefaultConnectionString(this IConfiguration configuration)
+        {
+            return configuration.GetConnectionString("DefaultConnection");
```

Comment:
```suggestion
            return configuration?.GetConnectionString("DefaultConnection");
```

To prevent a NRE (when not called as extension method). Maybe add a test for that too?

Reply:
```suggestion
            return configuration.GetConnectionString("DefaultConnection");
```

Actually, I'm not sure this is required, because even `configuration` will be `null`, the compiler will still link the ex. method properly, and `GetConnectionString` will in turn return `null`.
I'll add a new commit containing a test for this case.

Reply:
You are right! Smart compiler :smile:

Reply:
It's the one thing that might be a bit confusing about ex. methods, they feel like instance methods when they're essentially static ones.